### PR TITLE
Update fetch-oracle-data.sh

### PR DESCRIPTION
**Pull request by matiwinnetou**
- bug fixe
- added cryptoCompare
- refactoring

### CODE REVIEW

File: fetch-oracle-data.sh [hunk 1]
```diff
@@ -1,83 +1,97 @@
...
+echo "         \"value\": \"$(jq '.market_data.current_price.usd' <<< "$COIN_GECKO_ERGO_JSON")\","
 echo "         \"source\": \"coinGecko\""
 echo "       },"
 echo "       {"
-echo "         \"value\": \"$ERGOPOOLERGUSD\","
+echo "         \"value\": \"$(jq '.latest_price' <<< "$ERGO_POOL_ERG_USD_JSON")\","
 echo "         \"source\": \"ergoOracles\""
 echo "       }"
 echo "    ],"
 echo "    \"TSLA\": ["
```
Comment (matiwinnetou) [inline fetch-oracle-data.sh:89]: @mmahut I think it should be called: TSLAUSD

Comment (mmahut) [reply fetch-oracle-data.sh:89]: Is that stock traded in other currency too?

**Review by mmahut — APPROVED** [review]

File: fetch-oracle-data.sh (hunk 1 above)
Comment (mmahut) [reply fetch-oracle-data.sh:89]: I would rather preserve the ticker/name from the sources instead of renaming.

Comment (matiwinnetou) [reply fetch-oracle-data.sh:89]: I think it is most likely also traded in EUR: https://www.finanzen.net/aktien/tesla-aktie

**Pull request merged**